vars/jobs.groovy

```diff
@@ -39,6 +39,16 @@ def build_job(name, parameters, is_stable)
   return job
 }
 
+def getBuildUser() {
+  def user = currentBuild.rawBuild.getCause(Cause.UserIdCause)
+  if (!user?.trim()) {
+    return "Unknown"
+  }
+  else {
+    return user.getUserId()
+  }
```

Comment:
you can do it in one line something like:
return (!user?.trim()) ? "Unknown": user.getUserId()